apps/src/code-studio/initializeCodeMirror.js

```diff
@@ -38,17 +38,17 @@ const INVALID_COLOR = '#d00';
  * CodeMirror editor.
  * @param {!string|!Element} target - element or id of element to replace.
  * @param {!string} mode - editor syntax mode
- * @param {function} [callback] - onChange callback for editor
- * @param {booblen} [attachments] - whether to enable attachment uploading in
- *        this editor.
+ * @param {Object} options - misc optional arguments
+ * @param {function} [options.callback] - onChange callback for editor
+ * @param {boolean} [options.attachments] - whether to enable attachment
+ *        uploading in this editor.
+ * @param {???} [options.onUpdateLinting]
+ * @param {!string!Element} [options.preview] - element or id of element to
```

Comment:
```suggestion
 * @param {(string|Element)} [options.preview] - element or id of element to
```
I think this is the JSDoc you're looking for. ([source](http://usejsdoc.org/tags-type.html))

Reply:
d'oh, yes it is. Thanks!